projects/1.19/assets/caffeinefilled/caffeinefilled/lang/zh_cn.json

```
@@ -0,0 +1,74 @@
+  "advancements.kookie_coffee_advancement.descr": "喝下Kookie咖啡",
+  "advancements.kookie_coffee_advancement.title": "向Kookie.com.br致敬",
+  "advancements.never_sleep.descr": "喝下一杯特浓咖啡",
+  "advancements.never_sleep.title": "从未睡觉",
+  "advancements.pinkarella_special_advancement.descr": "喝下特别粉色天使",
+  "advancements.pinkarella_special_advancement.title": "向《屠宰马》致敬",
+  "block.caffeinefilled.andesite_counter": "安山岩台面",
+  "block.caffeinefilled.blackstone_counter": "黑石柜台",
+  "block.caffeinefilled.brick_counter": "砖砌台面",
+  "block.caffeinefilled.coffee_bean_bag": "袋装咖啡豆",
+  "block.caffeinefilled.coffee_berries_bag": "袋装咖啡果",
+  "block.caffeinefilled.coffee_bush_phase_0": "咖啡丛生长阶段 0",
+  "block.caffeinefilled.coffee_bush_phase_1": "咖啡丛生长阶段 0",
+  "block.caffeinefilled.coffee_bush_phase_2": "咖啡丛生长阶段 0",
+  "block.caffeinefilled.coffee_bush_phase_3": "咖啡丛生长阶段 0",
+  "block.caffeinefilled.deepslate_counter": "黑石柜台/深板岩柜台",
+  "block.caffeinefilled.diorite_counter": "闪长岩台面",
+  "block.caffeinefilled.granite_counter": "花岗岩台面",
+  "block.caffeinefilled.lemon_crate": "箱装柠檬",
+  "item.caffeinefilled.afogatto": "阿芙佳朵",
+  "item.caffeinefilled.americano": "美式咖啡",
+  "item.caffeinefilled.black_coffee": "黑咖啡",
+  "item.caffeinefilled.black_eye": "黑眼咖啡",
+  "item.caffeinefilled.ca_phe_sua_da": "越式冰咖啡",
+  "item.caffeinefilled.cafe_au_lait": "欧蕾咖啡",
+  "item.caffeinefilled.cafe_com_cheirinho": "威士忌黑咖啡",
+  "item.caffeinefilled.caffe_tobio": "Tobio咖啡",
+  "item.caffeinefilled.cappuccino": "卡布奇诺",
+  "item.caffeinefilled.carioca": "西爱西咖啡/里约热内卢咖啡",
+  "item.caffeinefilled.carioca_de_limao": "柠檬西爱西咖啡/柠檬里约热内卢咖啡",
+  "item.caffeinefilled.chocolate": "巧克力",
+  "item.caffeinefilled.chocolate_ice_cream_bowl": "巧克力冰淇淋",
+  "item.caffeinefilled.coffee_bean": "烤咖啡豆",
+  "item.caffeinefilled.coffee_berries": "咖啡果",
+  "item.caffeinefilled.coffee_ice_cream_bowl": "咖啡冰淇淋",
+  "item.caffeinefilled.cold_brew": " 冷萃咖啡",
+  "item.caffeinefilled.cortado": "可塔朵",
+  "item.caffeinefilled.cream": "奶油",
+  "item.caffeinefilled.doppio": "双份浓缩咖啡",
+  "item.caffeinefilled.egg_coffee": "蛋蜜咖啡",
+  "item.caffeinefilled.espresso": "意式浓缩咖啡",
+  "item.caffeinefilled.flat_white": "白咖啡",
+  "item.caffeinefilled.foam": "泡沫",
+  "item.caffeinefilled.galao": "Galão",
+  "item.caffeinefilled.glass_cup": "玻璃杯",
+  "item.caffeinefilled.irish_coffee": "爱尔兰咖啡",
+  "item.caffeinefilled.kookie_coffee": "Kookie咖啡",
+  "item.caffeinefilled.latte": "拿铁咖啡",
+  "item.caffeinefilled.lemon": "柠檬",
+  "item.caffeinefilled.lemon_ice_cream_bowl": "柠檬冰淇淋",
+  "item.caffeinefilled.lemon_juice_bottle": "柠檬汁瓶",
+  "item.caffeinefilled.long_black": "长黑咖啡",
+  "item.caffeinefilled.lungo": "淡浓缩咖啡/加长浓缩咖啡/廊构咖啡",
+  "item.caffeinefilled.macciato": "玛奇朵咖啡",
+  "item.caffeinefilled.mazagran": "马扎格兰冰咖啡",
+  "item.caffeinefilled.milk_bottle": "奶瓶",
+  "item.caffeinefilled.milk_ice_cream_bowl": "牛奶冰淇淋",
+  "item.caffeinefilled.mocha": "摩卡咖啡",
+  "item.caffeinefilled.pinkarella_special": "特别粉色天使",
```

```suggestion
  "item.caffeinefilled.pinkarella_special": "粉色天使特调",
```
貌似来自某个小马宝莉的同人游戏中的反派，没找到中文译名
这里改了进度说明也要改
https://slaughterhorse.fandom.com/wiki/Pinkarella